limiter.go

```diff
@@ -6,11 +6,10 @@ import (
 
 // SetRateLimiter sets the rate limiter
 func (self *Runtime) SetRateLimiter(limiter *rate.Limiter) {
```

Comment:
[opt/q] Is it worth adding a test that confirms that we won't panic with a nil limiter?